rzbrp100Sup/rzbrp100_channel.template

```diff
@@ -0,0 +1,152 @@
+
+#   Output
+
+record(bi, "$(P)CH$(CHANNEL):OUTPUT")
+{
+    field(DESC, "Channel $(CHANNEL) Output Status")
+    field(DTYP, "stream")
+    field(INP, "@rzbrp100.proto getOutput($(CHANNEL)) $(PORT)")
+    field(SCAN, "2 second")
```

Comment:
Bit slower than usual - is the device unable to cope with a higher poll rate? If so can you mention that on the wiki?

Reply:
Have reduced scan period to `1 second`.